pmd-html/src/test/resources/net/sourceforge/pmd/lang/html/ast/testdata/SimpleHtmlFile.txt

```diff
@@ -0,0 +1,22 @@
++- #document[]
+   +- #doctype[@Name = "html", @PublicId = "", @SystemId = ""]
+   +- #text[@Image = " ", @NormalizedText = " ", @Text = "
+"]
+   +- html[]
+   |  +- #text[@Image = " ", @NormalizedText = " ", @Text = "
```

Comment:
normalized text is the default in html - multiple whitespaces are combined together, newlines are ignored. I left it in, as otherwise you won't have access to the original text anymore....

Reply:
I meant that the tree dump thing is supposed to write every node on a single line, so that eg `@Text` should see its newline escaped and printed as `" \n "` or so

Reply:
Actually this seems to be fix on the pmd/7.0.x branch only - in master, the newlines are not normalized

Reply:
I've backported it into master: c224209d7fe12f0b7e2a1c34b25f6ee562f8addc
That should avoid some merge conflicts in the future.